src/sentry/grouping/ingest/hashing.py

```diff
@@ -239,22 +235,17 @@ def get_hash_values(
         secondary_hashes,
     )
 
-    all_hashes = CalculatedHashes(
-        hashes=extract_hashes(primary_hashes) + extract_hashes(secondary_hashes),
-        # We don't set a combo `variants` value here because one set of variants would/could
-        # partially or fully overwrite the other (it's a dictionary), and having variants from two
-        # different configs all mixed in together makes no sense.
+    return (
```

Comment:
is it necessary to return the third combined item here? can the callers just concatenate them?

Reply:
Good point. I will look into it.